pharmd-app/src/components/Inputs/SearchInput.js

```diff
@@ -6,28 +6,18 @@
  *        -- Create more generic search fields
```

Comment:
Is this already an issue that exists or is fixed?

Reply:
`* TODO:`
 *        -- Abstract the component: Potentially make this searchbox into a generic text field that takes in an icon.
 *        -- Use Tailwind classes
 *        -- Create more generic search fields